tools/windows/install-help/cal/stopservices.cpp

```diff
@@ -849,20 +849,20 @@ int uninstallServices(CustomActionData &data)
 #define NUM_SERVICES 4
     serviceDef services[NUM_SERVICES] = {
         serviceDef(agentService.c_str(), L"Datadog Agent", L"Send metrics to Datadog", agent_exe.c_str(),
-                   L"winmgmt\0\0", SERVICE_AUTO_START, data.FullyQualifiedUsername().c_str(), NULL),
+                   L"winmgmt\0\0", SERVICE_AUTO_START, NULL, NULL),
         serviceDef(traceService.c_str(), L"Datadog Trace Agent", L"Send tracing metrics to Datadog", trace_exe.c_str(),
-                   L"datadogagent\0\0", SERVICE_DEMAND_START, data.FullyQualifiedUsername().c_str(), NULL),
+                   L"datadogagent\0\0", SERVICE_DEMAND_START, NULL, NULL),
         serviceDef(processService.c_str(), L"Datadog Process Agent", L"Send process metrics to Datadog",
                    process_exe.c_str(), L"datadogagent\0\0", SERVICE_DEMAND_START, NULL, NULL),
         serviceDef(systemProbeService.c_str(), L"Datadog System Probe", L"Send network metrics to Datadog",
-                   sysprobe_exe.c_str(), data.npmPresent() ? probeDepsWithNPM : probeDepsNoNPM, SERVICE_DEMAND_START, NULL, NULL)
+                   sysprobe_exe.c_str(), L"\0\0", SERVICE_DEMAND_START, NULL, NULL)
```

Comment:
We don't need any of those values for uninstall. The code was incorrectly using an uninitialized CustomActionData instance. Since I modified that class it required me to modify this code in some way.

Looking again, it would be better to replace all of the `serviceDef` constructors with the one that only takes the service name.
For example
```
serviceDef(processService.c_str())
```